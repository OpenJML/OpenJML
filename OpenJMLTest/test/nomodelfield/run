#! /usr/bin/env bash

FILE="$BASH_SOURCE[0]"
DIR=`dirname "$FILE"`
cd "$DIR"

./runx |& diff - RUN.out
