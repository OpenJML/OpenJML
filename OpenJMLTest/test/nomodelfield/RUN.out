Exception in thread "main" java.lang.NoSuchMethodError: 'int P.`model$i()'
	at T.main(T.java:4)
